spec/features/comments_spec.rb

```diff
@@ -76,15 +76,22 @@
     end
   end
 
-  scenario 'Create', :js do
+  scenario 'Count comments', :js do
     login_as(user)
     visit debate_path(debate)
+    2.times do |time|
+      fill_in "comment-body-debate_#{debate.id}", with: "Have you thought about...#{time}?"
+      click_button 'Publish comment'
+      waiting_for_ajax
```

Comment:
Creo que esto no debería de ser necesario (y por tanto el módulo `WaitingForAjax `), si mueves `expect(page).not_to have_content 'No comments'` a esa posición: `capybara` ya espera a que las llamadas *ajax* en casi todas las ocasiones: https://github.com/jnicklas/capybara#asynchronous-javascript-ajax-and-friends